avm/res/db-for-my-sql/flexible-server/main.bicep

```
@@ -162,6 +165,16 @@ param diagnosticSettings diagnosticSettingType
 @description('Optional. Enable/Disable usage telemetry for module.')
 param enableTelemetry bool = true
 
+var standByAvailabilityZoneTable = {
+  Disabled: null
+  SameZone: availabilityZone
+  ZoneRedundant: highAvailabilityZone
+}
+
+var standByAvailabilityZone = contains(standByAvailabilityZoneTable, highAvailability)
+  ? standByAvailabilityZoneTable[highAvailability]
+  : null
```

Is that the same as 
```suggestion
var standByAvailabilityZone = standByAvailabilityZoneTable[?highAvailability]
```
? ... Rending the variable 'potentially' redundant (as it's quite short).